pinot-common/src/thrift/query.thrift

```diff
@@ -32,6 +32,8 @@ struct PinotQuery {
   11: optional map<string, string> queryOptions;
   12: optional bool explain;
   13: optional map<Expression, Expression> expressionOverrideHints;
+  14: optional bool showTables;
```

Comment:
Do we need to add it into `PinotQuery`? I don't think we need to send the request to server. Ideally we can get this info from the parsed `PlanNode` without constructing the `PinotQuery`

Reply:
I've removed it from `PinotQuery` now